packages/core/src/common/disposable.ts

```diff
@@ -47,6 +47,34 @@ Object.defineProperty(Disposable, 'NULL', {
     }
 });
 
+/**
+ * Utility for tracking a collection of Disposable objects.
+ *
+ * This utility provides a number of benefits over just using an array of
+ * Disposables:
+ *
+ * - the collection is auto-pruned when an element it contains is disposed by
+ * any code that has a reference to it
+ * - you can register to be notified when all elements in the collection have
+ * been disposed[1]
```

Comment:
```suggestion
 * been disposed [1]
```

Not sure cause usually we do not have a space when we actually have superscript but having no space here seems a bit odd to me.